src/services/etherspot.js

```diff
@@ -144,6 +175,133 @@ class EtherspotService {
     });
   }
 
+  clearTransactionsBatch() {
+    return this.sdk.clearGatewayBatch();
+  }
+
+  setTransactionsBatch(transactions: EthereumTransaction[]) {
```

Comment:
I found it difficult to rename properly so tried to follow our app flows. This is not executing transactions after all, but only putting to draft batch which is then needed to be estimated and submitted (2 more steps before it takes actual execution). That's why I thought that in this step we're actually just setting instead of executing, wdyt?